src/test/java/com/google/api/codegen/testsrc/libraryproto/library_gapic.yaml

```diff
@@ -260,6 +260,9 @@ interfaces:
               - title
       - print:
         - "That's all!"
+      - print:
+        - "series_uuid: %s"
+        - $resp.series_uuid
```

Comment:
Didn't we want to test nested fields? Should this be `$resp.series_uuid.something`?

Reply:
`series_uuid` is already a message nested in `PublishSeriesResponse`? But I updated the yaml to print `$resp.series_uuid.series_bytes`